### PR TITLE
Add support for BLS zipl

### LINKED ISSUE #2481: Missing support for ZIPL for s390x
## Problem description
When working on porting Fedora s390x images to KIWI, I discovered we do not support ZIPL directly and thus can't create s390x images that boot.

Fedora (and RHEL) use [ZIPL with Bootloader Spec to boot the system since Fedora 30](https://fedoraproject.org/wiki/Changes/BootLoaderSpecByDefault) and there is no `grub2-emu` for s390x available.

`zipl` bootloader should be supported for s390x, and it should support working with BLS configuration.

## OS and Software information

* KIWI version: 9.25.21
* Operating system host version: Fedora 40
* Operating system target version: Fedora 40
* Open Build Service version (N/A if not using OBS): N/A
* Koji version (N/A if not using Koji): N/A

### PR DESCRIPTION
**Pull request by schaefi**
Add support for

```xml
<bootloader name="zipl" .../>
```

to support BLS based zipl configuration.
This Fixes #2481

### CODE REVIEW

**Comment by schaefi** [comment]
I'm still testing the different boot disks for s390 which will need a bit of time

- [ ] SCSI disk 
- [ ] FBA (byte array) DASD disk
- [ ] DASD 4k block disk

**Comment by Conan-Kudo** [comment]
Can we make sure we have a test image with Rawhide s390x to validate this?

File: build-tests/s390/rawhide/test-image-disk/appliance.kiwi [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    </users>
+    <repository type="rpm-md" alias="kiwi-next-generation" priority="1">
+        <source path="obs://Virtualization:Appliances:Staging/Fedora_Rawhide"/>
+    </repository>
+    <repository type="rpm-md" alias="Fedora_Rawhide">
+        <source path="obs://Fedora:Rawhide/standard"/>
+    </repository>
+    <repository type="rpm-md" alias="Fedora_EPEL">
+        <source path="obs://Fedora:EPEL:8/standard"/>
+    </repository>
```
Comment (Conan-Kudo) [inline build-tests/s390/rawhide/test-image-disk/appliance.kiwi:44]: This needs to be dropped.

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

**Comment by schaefi** [comment]
Ok for me things are working now. Please find the integration tests here:

* https://build.opensuse.org/project/show/Virtualization:Appliances:Images:Testing_s390:rawhide

The Physical profile is for 4k DASD targets whereas the Virtual profile is for zFCP, FBA or virtual (e.g qemu) targets with 512b blocksize

I have tested on s390 itself but also in qemu. If you want to test in qemu please call

```
qemu-system-s390x -m 4096 -nographic -drive if=none,id=dr1,file=kiwi-test-image-disk.s390x-1.42.1-Virtual-Build10.5.qcow2 -device virtio-blk,drive=dr1,bootindex=1
```

Thanks

**Review by Conan-Kudo — APPROVED** [review]

**Pull request merged**